components/files/private/file-actions.jsx

```diff
@@ -0,0 +1,90 @@
+/* Copyright (c) 2015-present, salesforce.com, inc. All rights reserved */
+/* Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license */
+
+// # FileActions Component
+
+// Implements the [FileActions design pattern](https://www.lightningdesignsystem.com/components/files/) in React.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { FILES_ACTIONS } from '../../../utilities/constants';
+import Button from '../../button';
+import Icon from '../../icon';
+
+/**
+ * A carousel allows multiple pieces of featured content to occupy an allocated amount of space.
+ */
+const FileActions = (props) => {
+	const actions = (
+		<div className="slds-file__actions-menu">
+			<div className="slds-button-group" role="group">
+				{typeof props.onClickDownload === 'function' ? (
+					<Button
+						type="button"
+						variant="icon"
+						iconSize="x-small"
+						onClick={props.onClickDownload}
+						title="Download"
+					>
+						<Icon
+							assistiveText={{ label: 'download' }}
+							category="utility"
+							name="download"
+							size="xx-small"
+							inverse={!props.title}
+						/>
+					</Button>
+				) : null}
+				{typeof props.onClickMoreActions === 'function' ? (
```

Comment:
This should simply render `menuActionsDropdown` if present:
`{props.menuActionsDropdown}`